### ISSUE TITLE
master branch contains the screenplay pattern and not classic one 

### ISSUE THREAD

**Issue by mayank-kapoor** [serenity-bdd/serenity-cucumber-starter#33]
@wakaleo: master branch has the screenplay pattern rather the classic one.

Can you update the code.

**Comment by wakaleo** [comment]
That's intentional, but you don't have to use that approach if you don't want to - the action classes pattern is in the action-classes branch.

**Comment by traumatic** [comment]
Thanks. I think what is confusing is that the wiki page states the following 
"There are two versions to choose from. The master branch uses a more classic approach, using action classes and lightweight page objects, whereas the screenplay branch shows the same sample test implemented using Screenplay."

**Comment by wakaleo** [comment]
True, that needs updating. The Screenplay approach is now the preferred approach, but you can still use action classes or even plain page objects if it makes more sense for your context.

**Comment by traumatic** [comment]
Thanks John. The pom file in the action-classes branch shows screenplay dependencies. What are all the dependencies required if no screenplay dependencies are to be used? Is there an example you can refer to?

**Comment by traumatic** [comment]
Ok,seems that dropping them is sufficient.
